shared/test/test_assets.rb

```diff
@@ -186,10 +186,10 @@ def test_viewing_abusive_assets
     FilesApi.any_instance.unstub(:admin?)
 
     # teacher can view
-    FilesApi.any_instance.stubs(:teaces_student?).returns(true)
-    get(@assets, channel_id, asset_name)
-    assert @assets.last_response.successful?
-    FilesApi.any_instance.unstub(:teaces_student?)
+    FilesApi.any_instance.stubs(:teaches_student?).returns(true)
+    get(non_owner_assets, channel_id, asset_name)
+    assert non_owner_assets.last_response.successful?
+    FilesApi.any_instance.unstub(:teaches_student?)
```

Comment:
The reason this was working anyways, is we were looking at assets for which we were the owner (so we don't even hit teaches_student). By changing to non-owner assets, the test fails when we stub teaces_student, and then passes again when correcting the typo.